client/question/services/EventHandlerService.js

```diff
@@ -28,9 +59,15 @@ tie.factory('EventHandlerService', [
       createSessionPauseEvent: function(sessionId) {
         if (ServerHandlerService.doesServerExist()) {
           var data = {
```

Comment:
Optional, here and below: you could just directly declare

```
    var evt = {
      type: ...,
      data: {
        sessionId: sessionId,
        ...
      }
    }
```

or even do:

```
    currentEventBatch.push({
      type: ...,
      data: {
        sessionId: sessionId,
        ...
      }
    });
```

Reply:
Done.